container/build.sh

```diff
@@ -1,196 +0,0 @@
-# Copyright 2017 The Bazel Authors. All rights reserved.
```

Comment:
If we delete this script in the PR, I think we need to update the document as well?

Reply:
I'm not exactly sure which document you are referring to, the only one I found in this repo that referenced build.sh was the README which I have updated. Let me know if there is something I missed